.pre-commit-config.yaml

```diff
@@ -6,7 +6,7 @@
 
 repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.0.1
+    rev: v5.0.0
```

Comment:
Could I ask why you changed this? It's probably a good idea, but maybe let's do it in a different PR.

Reply:
You are right. There was a warning about a soon-to-be-deprecated naming and I applied it a little too fast. 
I have added a commit to remain at 4.0.1.